.pre-commit-config.yaml

```diff
@@ -45,7 +45,7 @@ repos:
       - id: check-toml
 
   - repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.3.0
```

Comment:
Maybe is better to update black in a separate PR and run it on all file as well :)

Reply:
To clarify:  Without it, the commit checks would not run.